### ISSUE TITLE
Комментарии к заявке

### ISSUE THREAD

**Issue by pvs9** [Danilovem/-.---#1]
1. Заявка должна быть файлом в репозитории с определенным названием (см. тестовый репозиторий).
2. Из функциональных требований не складывается представления о продукте. Чем он лучше обычного поиска на сайте.

**Comment by aakuptsov** [comment]
1. >>Для работы бота необходим Telegram.

Это вряд ли можно считать аппаратными требованиями. 

2. >>Компилятор для работы с Python

Питон, насколько мне известно, является языком интерпретируемым 

3. Из заявки неочевидно, какая именно информация будет предоставляться ботом, как она будет структурирована, как она попадает в бот, чем пользование ботом, действительно, отличается от поиска на сайте (см. комментарий выше). 

4. >>Написать бота.

Это, к сожалению, не достаточно подробный подход к планированию. Понятно, что процесс написания бота состоит из написания бота, нужно понять, на какие этапы его можно разбить. Это не просто наше пожелание: тебе самому будет проще работать по продуманному плану. 

Заявку нужно существенно доработать.

**Comment by aakuptsov** [comment]
На 25 декабря в репозитории в разделе Code так и не появился файл, названный и оформленный в соответствии с требованиями. Если файла не будет до НГ, зафиксирую пропуск первого дедлайна. Напоминаю, что заявка должна быть в отдельном файле, который должен быть назван и оформлен в соответствии с требованиями.

**Comment by Danilovem** [comment]
> 1. Заявка должна быть файлом в репозитории с определенным названием (см. тестовый репозиторий).
> 2. Из функциональных требований не складывается представления о продукте. Чем он лучше обычного поиска на сайте.

**Comment by Danilovem** [comment]
1. Сделал заявку в файле 
2. Написал в "Аппаратные требования" аппаратные требования Telegram, т.к. планируется бот в Telegram
3. Разбил написание бота на довольно большие этапы разработки
4. В пункте "Инструменты разработки" оставил язык Python

Мой продукт лучше, чем простой поиск на сайте тем, что с ботом можно пообщаться и получить после общения с ним необходимую информацию. Мне кажется, людям так будет приятнее искать информацию

**Comment by aakuptsov** [comment]
1. Заявку нужно дооформить (выделить **жирным** названия разделов) и исправить опечатки.
2. Добавить в заявку фамилию и имя и контакты. 
3. >>Изучить и классифицировать вопросы с почты сотрудников Центра Международной мобильности

Правильно ли я понимаю, что есть доступ к почте сотрудников и вопросов? 

4. >>Обработка сообщений на русском языке

Приведи, пожалуйста, пример запроса на русском языке. 

5. Как будет попадать информация с сайта в бот?

6. Как будет храниться информация в боте? Хардкодить эту информацию (хранить её в каких-то переменных или файлах) нельзя, поскольку изменения этой информации приведёт к необходимости вносить изменения при изменения информации.

**Comment by aakuptsov** [comment]
Ответов на комментарии пока нет.

**Comment by Danilovem** [comment]
3. Да, Вы правильно понимаете, что доступ к почте имеется
4.Привожу пример запроса: "Привет, Роберт! Как я могу поехать на мобильность?Какие мне нужны для этого документы?"
5 и 6 вопросы.  Я думал закидывать вручную информацию на сервер, и с этого сервера выгружать информацию в бот. Понимаю, что это не самый оптимальный вариант, но иначе проект входит еще более сложным.

В заявке добавил контактную информацию и выделил жирным разделы

**Comment by aakuptsov** [comment]
Возможно ли будет на защите добавить какой-то вопрос в бот через админку, а потом получить от бота на него ответ? Я сейчас не столько про точность ответа (это вопрос дискуссионный, понятно), сколько про саму возможность.

**Comment by Danilovem** [comment]
Да, будет возможность добавить вопрос через админку и получить на него ответ на защите

**Comment by aakuptsov** [comment]
Отлично. ЗАЧЁТ.
